src/i18n/en.js

```diff
@@ -183,6 +183,21 @@ You can avoid saturating your Database and possible removals. Do you need help?
     testAb: 'A/B Test Campaigns',
     title: 'Exclusive Features',
   },
+  faq: {
+    answer_1: `In your account, select Update Plan. Choose your Plan and fill your billing info. Click on "I have a promotion code", then enter your code and validate it.`,
+    answer_2: `Due to your amount of Contacts, you can acquire a Monthly High Volume Plan that will suit your business needs with a cheaper cost per Email.`,
+    answer_3: `Prepaid Plans' Credits don't have any type of expiration. You can use them for your sendings whenever you want. Also, Credits do accumulate.`,
+    answer_4: `You can pay using International credit cards from Mastercard, VISA or Amex. In Mexico, Argentina or Spain, bank transfers are allowed for purchases from US$77.`,
+    answer_5: `There is no minimum time to stay using Doppler. If you downgrade, only the current month will be charged. Need a contract? we can do it, but we wont ask for one.`,
+    answer_6: `If you choose a Pay in advance Plan, after meeting the agreed time, it will renew automatically for the same period you've previously selected.`,
+    question_1: `How to apply a Promotion Code to access a discount?`,
+    question_2: `Is there a Monthly Plan for more than 100,000 Contacts?`,
+    question_3: `Do Prepaid Plans' Credits have a deadline for their use?`,
+    question_4: `Which payment methods are available to purchase a Paid Plan?`,
+    question_5: `Is there a minimum contract of permanence when buying a Paid Plan?`,
+    question_6: `How does renewal works when paying 3, 6 or 12 months in advanced?`,
```

Comment:
```suggestion
    question_6: `How does renewal work when paying 3, 6, or 12 months in advanced?`,
```

Reply:
The texts were copied from here https://www.fromdoppler.com/es/precios-planes-mensuales/ 
do I still apply these changes?

Reply:
I do not know. @jmonacchi what do you think?

Reply:
This content needs a review with the Sales and probably the Administration team to understand if some of the FAQs change in the new system of Plans. Is it blocking for dev? If not, please go ahead and we can talk again about it later. Thanks!